src/video/wayland/SDL_waylandvideo.h

```diff
@@ -60,10 +51,7 @@ typedef struct {
     struct wl_shm *shm;
     struct wl_cursor_theme *cursor_theme;
     struct wl_pointer *pointer;
-    struct {
-        struct xdg_wm_base *xdg;
-        struct zxdg_shell_v6 *zxdg;
-    } shell;
+    struct xdg_wm_base *xdg_shell;
```

Comment:
We have to keep the `shell` struct to support multiple shell backends, e.g. https://github.com/libsdl-org/SDL/pull/4068.